src/buffer/manager/naivebis.rs

```diff
@@ -13,6 +13,41 @@ use crate::{
     log::manager::LogMgr,
 };
 
+// State Transition Diagram
```

Comment:
Is this diagram here?
Generally speaking, it is for any Buffer Manage, isn't it?